src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/EvidenceUploadService.java

```diff
@@ -99,7 +115,28 @@ private void addDocumentToCcd(String questionId, Long ccdCaseId, Document docume
 
         addNewCorDocumentToCaseDetails(questionId, document, caseDetails);
 
-        ccdService.updateCase(caseDetails.getData(), ccdCaseId, "uploadCorDocument", "SSCS - cor evidence uploaded", "Updated SSCS", idamTokens);
+        ccdService.updateCase(caseDetails.getData(), ccdCaseId, UPLOAD_COR_DOCUMENT, "SSCS - cor evidence uploaded", UPDATED_SSCS, idamTokens);
+    }
+
+    private void addSscsDocumentToCcd(Long ccdCaseId, Document document) {
+        log.info("Adding document to case {} ...",ccdCaseId);
+        IdamTokens idamTokens = idamService.getIdamTokens();
+        log.info("Got idam tokens ");
+
```

Comment:
We have already loaded the caseDetails earlier in this call probably needs to be passed through

Reply:
done